3rdparty/libprocess/src/process.cpp

```diff
@@ -843,7 +843,7 @@ void receive(Socket socket)
     if (f.isFailed()) {
       Try<Address> peer = socket.peer();
 
-      LOG(WARNING)
+      DLOG(WARNING)
```

Comment:
I think there is another occurence L1432

Reply:
there is yes, but in our kibana logs that one never seems to be hit so i left it as it is.

Reply:
We should change all occurrences now, it will make behavior more homogeneous

Reply:
done